mutiny-core/src/nodemanager.rs

```diff
@@ -157,6 +157,40 @@ impl From<Bolt11Invoice> for MutinyInvoice {
     }
 }
 
+impl From<MutinyInvoice> for PaymentInfo {
```

Comment:
not a huge deal but should we move `MutinyInvoice` to be defined somewhere else now that node manager doesn't even list invoices